code/modules/urist/modules/newtrading/NPC/NPC_types/teshari.dm

```diff
@@ -0,0 +1,72 @@
+/mob/living/simple_animal/hostile/npc/teshari
+	name = "teshari"
+	desc = "A teshari who decided to try their luck amongst the stars."
+	icon = 'code/modules/urist/modules/newtrading/NPC/npc.dmi'
+	icon_state = "Teshari_m"
+	emote_hear = list("coughs","sneezes","sniffs","clears their throat","whistles tunelessly","sighs deeply","yawns")
+	emote_see = list("shifts from side to side.","scratches their arm.","flaps their wings casually.","stares at at the ground aimlessly.","looks bored.","places their hands in their pockets.","stares at you with a blank expression.","scratches their beak.")
```

Comment:
I'm pretty sure Teshari don't have beaks.

Reply:
Yes, I'm pretty sure too.. 'scratches their muzzle/nose' could be better options